packages/address-book-controller/src/AddressBookController.ts

```diff
@@ -17,15 +21,17 @@ import type { Hex } from '@metamask/utils';
  * @property name - Nickname associated with this address
  * @property importTime - Data time when an account as created/imported
  */
-// This interface was created before this ESLint rule was added.
-// Convert to a `type` in a future major version.
-// eslint-disable-next-line @typescript-eslint/consistent-type-definitions
-export interface ContactEntry {
+export type ContactEntry = {
   address: string;
   name: string;
   importTime?: number;
-}
+};
 
+/**
+ * @type AddressType
+ *
```

Comment:
The `@type` tag is recognized by JSDoc, but it's ignored by TypeDoc and by the TypeScript language server (it can already see that this is a type because it knows this is an `enum`). So we can safely leave this tag off:

``` suggestion
```